.github/workflows/pull_request.yml

```diff
@@ -23,8 +23,10 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: '1.18'
+          cache: true
+      - run: go mod download -x
+      - run: go list ./...
```

Comment:
still need separate steps?

Reply:
I can drop this `go list ./..` stop, I don't think that adds anything. The separate `go mod download -x` step is nice because makes it easy to see how much time is spend downloading dependencies separate from the actual checks below. If things start to slow down we'll be able to see why without changing CI config.